### ISSUE TITLE
Create Neural Network for toy data

### ISSUE THREAD

**Issue by ssshathi** [ssshathi/PET-kaggle#1]
Implement a Neural Network from scratch, using only non-deep learning libraries

For this Issue to be closed, the following work items must be completed:

- [ ] It must implement both log loss and square loss correctly
- [ ] It must calculate gradients and update weights using back propagation correctly (Detailed Issue #2 )
- [ ] It must be able to take in toy data, train on it, and report that loss goes down (Create detailed issue)
- [ ] A separate script must be created to load data, input it to network, and plot metrics (Create detailed issue)
